packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -264,7 +264,7 @@ class IOSDevice extends Device {
           return null;
         }
 
-        printTrace('Application launched on the device. Attempting to forward ports.');
+        printTrace('Application launched on the device. Waiting for observatory port.');
```

Comment:
The previous wording was incorrect. Port forwarding is already happening in the background. Here we are just awating on the Future that will provide us with the port. It has no effect on port forwarding logic, which "Attempting to forward" implies.